test/visualisation/latexify.jl

```diff
@@ -121,35 +123,49 @@ let
     end
 
     # Latexify.@generate_test latexify(rn)
-    @test_broken latexify(rn) == replace(
-    raw"\begin{align*}
-    \varnothing &\xrightleftharpoons[d_{a}]{\frac{p_{a} B^{n}}{k^{n} + B^{n}}} \mathrm{A} \\
-    \varnothing &\xrightleftharpoons[d_{b}]{p_{b}} \mathrm{B} \\
-    3 \mathrm{B} &\xrightleftharpoons[r_{b}]{r_{a}} \mathrm{A}  
-    \end{align*}
-    ", "\r\n"=>"\n")
-
-    # Latexify.@generate_test latexify(rn, mathjax=false)
-    @test_broken latexify(rn, mathjax = false) == replace(
-    raw"\begin{align*}
-    \varnothing &\xrightleftharpoons[d_{a}]{\frac{p_{a} B^{n}}{k^{n} + B^{n}}} \mathrm{A} \\
-    \varnothing &\xrightleftharpoons[d_{b}]{p_{b}} \mathrm{B} \\
-    3 \mathrm{B} &\xrightleftharpoons[r_{b}]{r_{a}} \mathrm{A}  
-    \end{align*}
-    ", "\r\n"=>"\n")
+    @test latexify(rn) == replace(
+raw"\begin{align*}
+\varnothing &\xrightleftharpoons[d_{a}]{\frac{p_{a} B^{n}}{k^{n} + B^{n}}} \mathrm{A} \\
+\varnothing &\xrightleftharpoons[d_{b}]{p_{b}} \mathrm{B} \\
+3 \mathrm{B} &\xrightleftharpoons[r_{b}]{r_{a}} \mathrm{A}  
+ \end{align*}
+", "\r\n"=>"\n")
+
+    # Latexify.@generate_test latexify(rn, mathjax = false)
+    @test latexify(rn, mathjax = false) == replace(
+raw"\begin{align*}
+\varnothing &\xrightleftharpoons[d_{a}]{\frac{p_{a} B^{n}}{k^{n} + B^{n}}} \mathrm{A} \\
+\varnothing &\xrightleftharpoons[d_{b}]{p_{b}} \mathrm{B} \\
+3 \mathrm{B} &\xrightleftharpoons[r_{b}]{r_{a}} \mathrm{A}  
+ \end{align*}
+", "\r\n"=>"\n")
 end
 
 # Tests for system with parametric stoichiometry.
 let
     rn = @reaction_network begin
         p, 0 --> (m + n)*X
     end
-    
-    @test_broken latexify(rn) == replace(
-    raw"\begin{align*}
-    \varnothing &\xrightarrow{p} (m + n)\mathrm{X}  
-     \end{align*}
-    ", "\r\n"=>"\n")
```

Comment:
this just seem to be git getting confused. No tests have been moved (although tabs and empty liens were removed in places, which i presume is causing this).